AcademyApp/Extensions/UILabelExtensions.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UILabelExtensions.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 03/07/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+// MARK: - Extension of UILabel's UI setup
+
+extension UILabel {
+
+    // Method sets spacing between lines inside UILabel
+
+    func setLineHeight(lineHeight: CGFloat) {
```

Comment:
It would be better to have this as an extension of NSAttributedString instead of UILabel

Reply:
Updated to be extension  of NSMutableAttributedString.